docs/v1/use-cases.md

```diff
@@ -1,96 +1,143 @@
+3. Fail if service instances still exist for the broker
+
+## Catalog Publishing/Curation/Discovery
+
+* How are services identified: name, service name/id, plan name/id?
+* Who can see which services? (TODO: Include scope? Global/Namespaced)
+* Who can see which service instances? (TODO: Include scope? Global/Namespaced)
+* Which service instances are globally visible?
+* Catalog curation: which broker-provided services are globally visible?
+* Catalog curation: which namespaces can see which catalog services?
+* Does my catalog list not only my service types but also my instances?
+  * E.g., I have a dev cluster running on my local machine and I want to connect
+    my application with a database that exists in a test cluster. How do I find
+    that database instance?
+
+## Requesting Services (Consumer)
+
+* As a User, how do I ask for a new service from the Catalog?
+* As a User, how do I ask for access to an existing service?
+* How does the catalog support multiple consumers in different Kubernetes
+  namespaces of the same instance of a service?
+
+## Provisioning a Service Instance
+
+* As a Broker operator, I want to control the number of instances of my Service,
+  so that I can control resource footprint of my Service.
+* As an implementer of a service provider on k8s, where may I provision
+  Kubernetes resources so that I may provide the requested service.
+* As an implementer of a service provider on k8s, what credentials should I use
+  to provision Kubernetes resources so that I may provide the requested service.
+
+## Binding to a Service Instance
+
+* As a Broker operator, I want to control the number of bindings to a Service
+  Instance so that I may provide limits for services (e.g. free plan with 3
+  bindings). (TODO: Do we care?)
+* As a user of a service instance, I want a predictable set of Kubernetes
+  resources (Secrets, ConfigMap) created after binding, so that I know how to
+  configure my application to use the Service Instance.
+* As a service operator, I want to be able to discover what applications are
+  bound to services I am responsible for, so that I may operate the service
+  properly.
+
+## Using/Consuming a Service Instance
+
+* What is the unit of consumption of a service? Namespace? Pod? Something else?
+  (brian to comment)
+* As a User consuming a Service, I need to be able to understand the structure
+  of the resources that I get, so that I can configure my application
+  appropriately.
+* As a User, I want to be able to understand the relationship between a Secret
+  and Service Instance, so that I can properly configure my application (e.g.
+  app connecting to sharded database).
+
+### Consuming bound services
+
+Consumers of a service provisioned through the Service Catalog should be able
+to access credentials for the new Service Instance using standard Kubernetes
+mechanisms.
+
+1. A Secret maintains a 1:1 relationship with a Service Instance Binding
```

Comment:
To be clear... in this "1:1" relationship, a Binding is on one side, what's on the other? A secret or an app?  I'm guessing a secret, which means the secret doesn't "maintain" the relationship (its not the link) - so perhaps:  There is a 1:1 relationship between Secrets and Service Instance Bindings.

So we think there will be other things related to Bindings? I'm asking because if there's always one secret per binding and nothing else, can we combine the two and just have secrets, no binding at all?